### DISCUSSION TITLE
ArcWarden Aghanim Rework

### DISCUSSION THREAD

**Discussion by LuckyCat** [ValveSoftware/Dota2-Gameplay#14904] [Ideas]
Flux can be cast on allies or self. Heals for the amount instead of damage. Also gives +40% status resistance to an ally or -40% status resistance to an enemy.

**Comment by noxiouswire** [comment]
Unlock endless possibilities and seize the opportunity to make your mark through this captivating game. [geometry dash meltdown](https://geometrydash-meltdown.co/)
